packages/@aws-cdk/aws-certificatemanager/lib/dns-validated-certificate.ts

```diff
@@ -122,6 +122,18 @@ export class DnsValidatedCertificate extends CertificateBase implements ICertifi
     requestorFunction.addToRolePolicy(new iam.PolicyStatement({
       actions: ['route53:changeResourceRecordSets'],
```

Comment:
I know this wasn't changed, but is this the correct casing for this action? The `c` is lowercase but elsewhere it is uppercase.

Reply:
IAM actions are case insensitive, but I can update if you want (https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_action.html)

Reply:
🤯